### PR TITLE
Added sections on the landing page

### PR DESCRIPTION
**Pull request by NB134**
I tried doing the same things that i did on the bundly/dash repo code, but i have ran into some problems.
I have written the same code in both the repository files. But in  dash-beta the font is not being applied to the page.
below is the image from bundly/dash repo's page
image

and here is the bundly/dash-beta repo's page
image
 
the font is not being applied to the page, also there are some classes which do not exist in login.scss but their style was being applied to the elements in the bundly/dash repo but it is not being applied in this repo.

Like the h1 element which contains the heading "Bundly" has classes "d-flex" ,"justify-content-center" ,"display-3" but they do not exist in the login.scss and there is no other stylesheet imported in login.js

upon inspecting the h1 element the styles form the above classes are being applied from scss files from coreui in the node modules, i do not know how to set it up for the dash-beta repo.
image

How should i proceed to fix it?

### CODE REVIEW

**Comment by sauravhiremath** [comment]
Thanks for opening a PR here. I have added some minor fixes:

* Allow h1 heading renders
* Root styling applied to the landing page
* #2 should fix the serif fonts in the description

Lemme know if this is fine and feel free to tune it yourself. We will want to land this feature soon.

So, please remove the feature from Draft when it is ready

**Comment by NB134** [comment]
thank you for helping me, the color and font-size is different than the original landing section, i was able to adjust the font-size similar to the original but the primarytext color of index.js is different than the one that was used for the heading.
image
the color of current heading is #263238 whereas in the previous heading it was #3c4b64. Should i keep the colour same if not how should i proceed to change it. Also the how to change the color of secondary text as it is written without the typography element

**Comment by sauravhiremath** [comment]
@NB134 the font fix PR is merged to master. Could you sync your head with the remote master and check if the sans serif font issue is fixed?

Also, you can import colors and use the colors from there, instead of hardcoding the values yourself

```javascript
import { colors } from '@material-ui/core'`;
colors.blueGrey[50]; // Or you can just use the default text color (primary) which is colors.blueGrey[600]
```
You can remove the draft status once the above fix is done, so it can be merged to master.

Apologies for the late response! was caught up in some other work

**Comment by NB134** [comment]
I have changed the colors using value of color.blueGrey[500] in inline style attribute, but for the co-ordinates i had to change color in css as they are pseudo elements. Kindly check it on https://dash-beta.vercel.app/.

**Review by sauravhiremath — APPROVED** [review]
LGTM

**Comment by kartikcho** [comment]
I'm seeing changes from #2 in this PR, are you sure you rebased from the correct master @NB134 ? You'll have to rebase from origin/master, not local master.

File: src/components/Logo.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ const Logo = (props) => {
   return (
     <img
       alt="Logo"
-      src="/static/logo.svg"
+      src="/static/logowhite.png"
+      style={{width: '42px'}}
```
Comment (kartikcho) [inline src/components/Logo.js:8]: example, here this has already been committed to master.

https://github.com/bundly/dash-beta/blob/9718d5682e94f5da5e5c3d5ee3cf66dcece6defa/src/components/Logo.js#L7-L9

**Comment by kartikcho** [comment]
I'll be squashing this PR to get the extra commits kinked out in master, if it doesn't fix itself, I'll revert this PR and you'll have to make a new one @NB134.

File: src/components/Logo.js (hunk 1 above)
Comment (sauravhiremath) [reply src/components/Logo.js:8]: Closed as merged

**Pull request merged**